app.py

```diff
@@ -23,7 +26,11 @@ def send_inital_message():
 
 @app.route("/send-reminders", methods=["POST"])
 def send_reminder():
-    return jsonify(send_messages())
+    indicator = request.form['indicator']
+    message = request.form['message']
+    print(indicator, message)
+    send_messages(indicator, message)
+    return "message sent successfuly"
```

Comment:
A view should return a Flask Response object not a str
the quick way to do it is with jsonify

Reply:
will update!